src/components/GlobalLeaderboards/TopAccounts.tsx

```diff
@@ -0,0 +1,154 @@
+import { useState } from "react";
+import { BigNumber } from "ethers";
+import classnames from "classnames";
+import { t } from "@lingui/macro";
+import { formatAmount, formatUsd } from "lib/numbers";
+import { useDebounce } from "lib/useDebounce";
+import Pagination from "components/Pagination/Pagination";
+import TableFilterSearch from "components/TableFilterSearch";
+import Table from "components/Table";
+import { useLeaderboardContext } from "./Context";
+import { USD_DECIMALS } from "lib/legacy";
+import { TableCell, TableHeader } from "components/Table/types";
+import { TopAccountsRow, formatDelta } from "domain/synthetics/leaderboards";
+import AddressView from "components/AddressView";
+import Tooltip from "components/Tooltip/Tooltip";
+
+const parseRow = (s: TopAccountsRow): Record<string, TableCell> => ({
+  id: s.id,
+  rank: s.rank + 1,
+  account: {
+    value: "",
+    render: (_, breakpoint) => (
+      s.account && (
+        <AddressView
+          address={ s.account }
+          ensName={ s.ensName }
+          avatarUrl={ s.avatarUrl }
+          breakpoint={ breakpoint }
+          size={ 24 }
+        />
+      )
+    ),
+  },
+  absPnl: {
+    value: "",
+    render: () => (
+      <Tooltip
+        handle={ formatDelta(s.absPnl, { signed: true, prefix: "$" }) }
+        position="center-top"
+        renderContent={
+          () => (
+            <div>
+              <p>{ `${ t`RPnL` }: ${ formatUsd(s.rPnl) }` }</p>
+              <p>{ `${ t`UPnL` }: ${ formatUsd(s.uPnl) }` }</p>
+            </div>
+          )
+        }
+      />
+    ),
+    className: classnames(
+      s.absPnl.isNegative() ? "negative" : "positive",
+      "leaderboard-pnl-abs"
+    ),
+  },
+  relPnl: {
+    value: "",
+    render: () => (
+      <Tooltip
+        handle={ formatDelta(s.relPnl.mul(BigNumber.from(100)), { signed: true, postfix: "%" }) }
+        position="center-top"
+        renderContent={
+          () => (
+            <div>
+              <p>{ `${ t`Max Collateral` }: ${ formatUsd(s.maxCollateral) }` }</p>
+            </div>
+          )
+        }
+      />
+    ),
+    className: classnames(
+      s.relPnl.isNegative() ? "negative" : "positive",
+      "leaderboard-pnl-rel"
+    )
+  },
+  size: {
+    value: formatUsd(s.size) || "",
+    className: "leaderboard-size"
+  },
+  leverage: {
+    value: `${formatAmount(s.leverage, USD_DECIMALS, 2)}x`,
+    className: "leaderboard-leverage"
+  },
+  perf: {
+    value: `${ s.wins.toNumber() }/${ s.losses.toNumber() }`,
+    className: "text-right",
+  }
+});
+
+export default function TopAccounts() {
+  const perPage = 15;
+  const [page, setPage] = useState(1);
+  const [search, setSearch] = useState("");
+  const term = useDebounce(search, 300);
+  const { topAccounts, topAccountsHeaderClick } = useLeaderboardContext();
+  const { isLoading, error } = topAccounts;
+  const filteredStats = topAccounts.data.filter(a => a.account.indexOf(term.toLowerCase()) >= 0);
```

Comment:
hm, why you define isLoading and error as separated variables but not define data?)